src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/TracePoint.kt

```diff
@@ -24,6 +25,12 @@ data class Trace(
     }
 }
 
+private val threadDefaultValues = listOf("true", "false", "null", "null", "-1", "")
+private val threadParamNames = listOf("start", "isDaemon", "contextClassLoader", "name", "priority", "block")
+private const val THREAD_CLASS_NAME = "kotlin.concurrent.ThreadsKt"
+internal const val THREAD_FUN_NAME = "thread"
```

Comment:
Let's group all this information into one class, something like this:

```kotlin
private class FunctionInfo(
    val className: String, 
    val methodName: String, 
    val parameterNames: List<String>, 
    val defaultValues: List<String>,
)
```
 
and then add: 
```kotlin
val threadFunctionInfo = FunctionInfo(...)
```

The name `FunctionInfo` is debatable, if you have better name in mind please use it.